core/modules/views_ui/views_ui.admin.inc

```diff
@@ -4401,21 +4403,21 @@ function views_ui_admin_settings_basic() {
     }
   }
 
-  // This is not currently a fieldset but we may want it to be later,
-  // so this will make it easier to change if we do.
+  // This is not currently a fieldset but we may want it to be later, so this
+  // will make it easier to change if we do.
   $form['basic'] = array();
 
   $form['basic']['show_master_display'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Always show the master display'),
-    '#description' => t('Advanced users of views may choose to see the master (i.e. default) display.'),
+    '#title' => t('Always show the <em>Master</em> display'),
+    '#description' => t('Show the <em>Master</em> (i.e. default) display, which is otherwise hidden by default.'),
```

Comment:
Not sure we need this "otherwise" here, not in addition to the "by default". One or the other is sufficient.

Reply:
Done!